COLLECTION.md

```diff
@@ -6,6 +6,7 @@
 
 |  #  | Brand              | Type        | Name                        | Origin      | Flavour                     |
 |-----|--------------------|-------------|-----------------------------|-------------|-----------------------------|
+| 142 | [Jia ming]         | [Green]     | [West Lake Dragon Well]     | [China]     | Green tea                   |
```

Comment:
In China, many shops don't have their own website at all, they do business via taobao (a ebay-like website), wechat, friends group, but not a website. That's a too old way for them.